test/e2e_node/cpu_manager_metrics_test.go

```diff
@@ -91,11 +91,12 @@ var _ = SIGDescribe("CPU Manager Metrics", framework.WithSerial(), feature.CPUMa
 		ginkgo.AfterEach(func(ctx context.Context) {
 			if testPod != nil {
 				deletePodSyncByName(ctx, f, testPod.Name)
+				waitForContainerRemoval(ctx, testPod.Spec.Containers[0].Name, testPod.Name, testPod.Namespace)
 			}
 			updateKubeletConfig(ctx, f, oldCfg, true)
 		})
 
-		ginkgo.It("should report zero pinning counters after a fresh restart", func(ctx context.Context) {
+		ginkgo.It("[pinning] should report zero pinning counters after a fresh restart", func(ctx context.Context) {
```

Comment:
How do you plan to use the `[pinning]` tag?

Reply:
It's the easiest and simplest way I know to run a subset of tests without using crazy regexes using `gingko --focus`

Reply:
My question is whether you want to use it just locally or also in test-infra? We prefer to switch to the ginkgo tags, and we started moving there. Ginkgo tags make it easier to query which tests to run

Reply:
If it is just for local debugging - I would question the value of it being added permanently. Is it easy to understand what pinning vs. alignment means? I have difficulty understanding it

Reply:
I can move to labels if it helps

Reply:
actually removed